src/components/form/form.style.js

```diff
@@ -16,14 +19,49 @@ export const StyledAdditionalFormAction = styled.div`
       display: inline-block;
     `}
 
-    ${StyledButton} {
+    ${applyButtonLeftMargin}
+
+    && ${StyledButton} {
+      margin-left: 0px;
       &:first-child {
         margin-left: 0;
       }
     }
   `}
 `;
 
+export const StyledResponsiveFooterWrapper = styled.div`
+  margin: 0 auto;
+  max-width: inherit;
+  min-width: inherit;
+  padding: 0;
+  align-items: center;
+  display: flex;
+  margin-top: 20px;
+  ${({ borderWidth }) => borderWidth && css`
+    border-width: ${borderWidth} !important;
```

Comment:
Is there an alternative to using `!important` here? Maybe [this StyledComponents suggestion](https://www.styled-components.com/docs/faqs#how-can-i-override-styles-with-higher-specificity) could be used?

> The way to override styles with a high specificity is to simply increase the specificity of your own styles. This could be done using `!important`, but that's error prone and generally not a good idea. We recommend the following technique ... Each `&` gets replaced with the generated class.

Reply:
Good point I'll use `&&` I wrongly assumed it had to be applied to a styled-component but it clearly works for specific properties as well 👍

Reply:
In fact I  ended up having to use `&&&&` 😱

Reply:
Wow, that's quite specific! :sweat_smile: Thanks for changing this :thumbsup: